test/jasmine/tests/click_test.js

```diff
@@ -47,7 +47,7 @@ describe('Test click interactions:', function() {
             setTimeout(function() {
                 click(x, y);
                 setTimeout(function() { resolve(); }, DBLCLICKDELAY / 2);
-            }, DBLCLICKDELAY / 4);
+            }, DBLCLICKDELAY / 2);
```

Comment:
Possibly. Though there may be cases where there never is a `Plotly.plot` call after the click.